source/Octopus.Tentacle.Contracts/CancelScriptCommandV2.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Octopus.Tentacle.Contracts
+{
+    public class CancelScriptCommandV2
```

Comment:
We version the whole interfaces, not individual operations, so the number of classes under `Octopus.Tentacle.Contracts` can grow quickly.  Did you consider putting all `IScriptServiceV2` classes in a subfolder?

Reply:
Yeh good idea 👍